### DISCUSSION TITLE
Is board has specified power limit?

### DISCUSSION THREAD

**Discussion by cpp0xc0ffeeee** [tjko/fanpico#38] [Q&A] [answered]
I want to connect few 12038 fans, but I'm worry the current is too high for fanpico.

**Comment by tjko** [comment, marked as answer]
Each of the (motherboard) inputs (MBFAN connectors) should handle at least 1.5A.   Which often is what the motherboard fan outputs are rated at.

While the AUX power connector has 3A poly-fuse (one probably could get away with slightly higher rated fuse).

For higher current fans, best probably supply power for fans 1 through 4, via the mbfan connectors, and then use the AUX for fans 5 through 8...

**Reply by siodor** [reply]
Hello @tjko 
Could this power limit please be written into the README.md?
Greetings
Theo

**Reply by tjko** [reply]
Sure, that's a good idea (maybe in the board specific README.md, as I'm working on new variant with different characteristics).

Btw, currently the "User Manual" documents the power limits already (https://github.com/tjko/fanpico/wiki/FanPico-User-Manual)
